### ISSUE TITLE
Only 1 peer of the network is shown

### ISSUE THREAD

**Issue by c-geek** [ucoin-io/sakia#343]
I've cleared my ~/.config/sakia folder and restarted Sakia, and for 20 minutes now I only see 1 node:

![image](https://cloud.githubusercontent.com/assets/969136/12261284/5573b6a6-b920-11e5-8b72-be6415e3cd4b.png)

**Comment by galuel** [comment]
And what about refreshing transactions !? It is blocked at 99%, I launched 0.11.14 (with 0.11.13 label...) with ./sakia -d (log option) and put the terminal in screen capture : 

![sakia](https://cloud.githubusercontent.com/assets/5245398/12261434/5cf1b9f4-b921-11e5-843d-0a48e930f681.png)

**Comment by Insoleet** [comment]
In the logs I can see many errors `Incorrect leaf reply` coming from here : 

https://github.com/ucoin-io/sakia/blob/master/src/sakia/core/net/node.py#L472

It's because you removed the `raw` peer document from the data since 0.13...
http://metab.ucoin.io/network/peering/peers?leaf=DE0578CCBCD462C2ED6B8F8F773A542CDCCFAA82

**Comment by c-geek** [comment]
I've added back locally (`metab.ucoin.io:9201` node) the `raw` field on the leaf, so the bug won't show up for the people with my node as root node.

So there is no more hurry right now.

Now about this `raw` field: I would prefer to remove it. What do you think?

**Comment by c-geek** [comment]
Thinking about it again, I think I should better make the `raw` field stay in the answer. BMA is a contract I've made with Sakia and other clients, theoretically I should never remove fields I've added nor change their field types. Of course type change already happened, but this should no more occur from ucoin@0.13.

The contract is defined by these 2 files:

* https://github.com/ucoin-io/ucoin/blob/master/app/lib/streams/bma.js#L68-L107
* https://github.com/ucoin-io/ucoin/blob/master/app/lib/streams/dtos.js

and I should no more change it, or just for adding fields.

If I want cleaner responses, I should just make another API.
